src/apis/study-room/study-room-info.ts

```diff
@@ -0,0 +1,16 @@
+import { http } from '../api';
+
+type Params = { studyId: number };
+type Response = {
+  id: number;
+  name: string;
+  intro: string;
+  step: string;
+  participantCount: number;
+  updateAt: string;
+};
+
+export const studyRoomInfo = async ({ studyId }: Params): Promise<Response> => {
+  const response = await http.get(`/studyrooms/${studyId}`);
+  return response.data;
```

Comment:
취향이지만 요렇게 표현해줘도 좋을 것 같네요~
```suggestion
  const { data } = await http.get(`/studyrooms/${studyId}`);
  
  return data;
```